### DISCUSSION TITLE
start with fragment file, peaks calling is too slow

### DISCUSSION THREAD

**Discussion by 0106WeiWeiDeng** [stuart-lab/signac#1272] [Q&A]
hello, 

I only have a fragment as input, and I follow the cmds from [this discussion](https://github.com/stuart-lab/signac/discussions/1025#discussioncomment-2342170). In step two peaks calling, it have been running for 23 hours (it still keep running now), how to speed up the peaks calling step?

**Comment by timoast** [comment]
Can you open an issue and show the full code and sessionInfo() output instead?
